geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -32,20 +51,30 @@ <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
                         {{ image_info.image|iiif_image:"size:width=640" }} 640w,
                         {{ image_info.image|iiif_image:"size:width=1000" }} 1000w">
                 </div>
-                <div class="transcription">
-                    {% if forloop.first %}<h3>Transcription</h3>{% endif %}
-                    {# display transcription in chunks by index #}
-                    <div>{{ document.digital_editions.0.content.html|index:forloop.counter0|h1_to_h3|safe }}</div>
+                <div class="transcription-panel">
+                    {% if forloop.first %}
+                        <h3>Transcription</h3>
+                        <span data-transcription-target="editionFullLabel" class="current-transcription">{{ document.digital_editions.0.display|safe }}</span>
+                    {% endif %}
+
+                    <div class="editions">
+                        {# display transcription in chunks by index #}
+                        {% for edition in document.digital_editions.all %}
+                            <div class="transcription" id="ed-{{ edition.pk }}" data-label="{{ edition.display|safe }}">
+                                {{ edition.content.html|index:forloop.parentloop.counter0|h1_to_h3|safe }}
+                            </div>
+                        {% endfor %}
+                    </div>
```

Comment:
I had been planning to leave this out for the mvp version, but it looks like it wasn't too difficult to add.

I do notice that the text is shifting vertically when switching between records that have longer/shorter citations. Can we find an easy solution to make the height match somehow? Should we always allow for two lines?

Reply:
Used some CSS to always make this at least two lines' height, let me know if this solution doesn't work!